ui/src/cloud/utils/reporting.ts

```diff
@@ -31,21 +31,14 @@ export const updateReportingContext = (properties: KeyValue) => {
   reportingTags = {...reportingTags, ...properties}
 }
 
-export const reportEvent = ({
-  timestamp = toNano(Date.now()),
-  measurement,
-  fields,
-  tags,
-}: Point) => {
+const pooledEvent = ({timestamp, measurement, fields, tags}: Point) => {
   if (isEmpty(fields)) {
     fields = {source: 'ui'}
   }
 
-  fireEvent(measurement, {...reportingTags, ...tags})
-
   reportingPoints.push({
-    measurement,
-    tags: {...reportingTags, ...tags},
+    measurement: 'UI Event', // artifact of sharing the tools cluster
+    tags: {...reportingTags, ...tags, event: measurement},
     fields,
     timestamp,
```

Comment:
does this need to be converted to nanoseconds?

Reply:
I will do this next !